assets/components/tinymcerte/js/mgr/tinymcerte.js

```diff
@@ -63,7 +63,7 @@ Ext.extend(TinyMCERTE.Tiny,Ext.Component,{
     ,loadBrowser: function(field_name, url, type, win) {
         tinyMCE.activeEditor.windowManager.open({
             title: "MODX Resource Browser",
-            url: MODx.config['manager_url'] + 'index.php?a=' + MODx.action['browser'] + '&source=' + MODx.config['default_media_source'],
+            url: MODx.config['manager_url'] + 'index.php?a=' + MODx.action['browser'] + '&source=' + MODx.config['default_media_source'] + (MODx.ctx ? ('&ctx=' + MODx.ctx) : ''),
```

Comment:
In fact there is no need to check for `MODx.ctx` since it's always defined by the [controller](https://github.com/modxcms/revolution/blob/2.x/manager/controllers/default/browser/index.class.php#L30)